freetextresponse/private/view.js

```diff
@@ -0,0 +1,47 @@
+function FreetextresponseView(runtime, element) {
+    'use strict';
+
+    var $ = window.jQuery;
+    var $element = $(element);
+    var buttonSubmit = $element.find('.check.Submit');
+    var textareaParent = $element.find('.student_answer').parent();
+    var usedAttemptsFeedback = $element.find('.action .used_attempts_feedback');
+    var problemProgress = $element.find('.problem-progress');
+    var submitParent = $element.find('.Submit').parent();
+    var wordCountError = $element.find('.word-count-error');
+    var url = runtime.handlerUrl(element, 'submit');
+
+    buttonSubmit.on('click', function () {
+        console.log('submit button was pressed!');
+        buttonSubmit.text('Checking...');
+        runtime.notify('submit', {
+            message: 'Submitting...',
+            state: 'start',
```

Comment:
Just to point it out... since this JS gets compiled, we can include trailing commas in Javascript (still not in JSON, though).